server/docs.js

```diff
@@ -16,7 +16,7 @@ const supportedTypes = new Set(['document', 'spreadsheet', 'text/html'])
 exports.cleanName = (name = '') => {
   return name
     .trim()
-    .replace(/^(\d+[-–—_\s]*)([^\d\/\-^\s]+)/, '$2') // remove leading numbers and delimiters
+    .replace(/^(\d+[-–—_\s]*)([^\d\\/\-^\s]+)/, '$2') // remove leading numbers and delimiters
```

Comment:
unrelated but this was highlighted when I ran `npm run lint` as recommended in description. I'm happy to remove this and put in a separate PR